CMakeLists.txt

```diff
@@ -100,7 +100,14 @@ if(CMAKE_USE_PTHREADS_INIT)
   set(PROTOBUF_CFLAGS "${PROTOBUF_CFLAGS} -pthread")
   set(PROTOBUF_CXXFLAGS "${PROTOBUF_CXXFLAGS} -pthread")
 endif()
-include(protobuf)
+
+if (DOWNLOAD_PROTOBUF)
```

Comment:
Could you make it an option:
e.g. option(LIB_PROTO_MUTATOR_WITH_ASAN "Enable address sanitizer" OFF) on line 21